src/App.tsx

```diff
@@ -15,71 +17,76 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+interface State {
+  product: string | null
+}
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    product: 'Jam',
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  render() {
+    const { product } = this.state;
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+    return (
+      <main className="section container">
+        <h1 className="title is-flex is-align-items-center">
+          {product ? `${product} is selected` : 'No goods selected'}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+          {product && (
+            // eslint-disable-next-line jsx-a11y/control-has-associated-label
             <button
-              data-cy="AddButton"
+              data-cy="ClearButton"
               type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+              className="delete ml-3"
+              onClick={() => this.setState({ product: null })}
```

Comment:
Move this to the separate fucntion
```suggestion
              onClick={handleCleat}
```